debian/tests/include-z3-test

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
```

Comment:
The builds that build and run examples are supposed to cover this test.

Reply:
This is done in the installed package as autopkgtest, not as build-time test.
This is useful to detect if something in the toolchain breaks the test of the package, without having to rebuild it by hand